docs/sdk/migrations/datil.md

```diff
@@ -0,0 +1,182 @@
+- A new Lit test token has been deployed for the testnet, and previous Lit test token balances from Chronicle will **not** be carried over.
+  - The new Lit test token address, deployed on Chronicle Vesuvius Testnet, is: `0x7FC23B31E9f262329cbc83c293a93b4e7176807E`
+  - Lit test tokens can be received from [this faucet](https://datil-dev-faucet.vercel.app/)
+
+### New Network, New PKPs
+
+Because existing PKPs were minted on the old Chronicle chain, migrating from `cayenne` to `datil-dev` requires re-minting of all PKPs. This also means transferring ownership of assets owned by PKPs minted on Chronicle, to the newly minted ones on Chronicle Vesuvius.
+
+To reduce the friction of migrating PKPs to Datil, we've written a [migration script](https://github.com/LIT-Protocol/developer-guides-code/tree/wyatt/pkp-migration-script/pkp-migration/nodejs) that will take a list of PKP public keys, fetch their configured Auth Methods and Scopes, and mint new PKPs on a target Lit Network, setting the same Auth Methods and Scopes for each PKP.
+
+:::note
+The migration script **will not** handle migration of any assets the existing PKPs own such as tokens. Assets held by existing PKPs will need to be manually transferred to a new PKP's Ethereum address (or some another address of your choosing) using a blockchain transaction.
+
+Additionally, the newly minted PKPs on the target Lit network will have new Ethereum addresses, so anything that uses the existing PKP's Ethereum address for things like permissions, will need to be manually updated to us the new PKP's Ethereum address.
+:::
+
+### Encrypted Data
+
+Because each Lit network undergoes it's own Distributed Key Generation (DKG), and therefore has it's own BLS root key, any encrypted data on one Lit network is **not** able to be decrypted using a different Lit network. For example, data encrypted using `cayenne`'s public BLS key, will **not** be able to be decrypted using the `datil-dev` network.
+
+Each Lit network has it's own BLS root key that's used for encrypting data. To migrate existing encrypted data, you will need to decrypt it using the Lit network it was encrypted with. Then re-encrypt it using one of the new Datil networks.
+
+There were no API changes made to the SDK for encrypting and decrypting, so you're existing code should work with Datil. See [here](../../sdk/access-control/quick-start#performing-encryption) for a guide on encrypting data, and [here](../../sdk/access-control/quick-start#performing-decryption) for decrypting.
+
+## How to Connect to a Datil Network
+
+:::note
+Currently only the `datil-dev` network is live and available to connect to. This guide will be updated when the other Datil networks come online.
+:::
+
+The only code changes required to make use of the new Datil networks are as follows:
+
+- Upgrade the Lit packages to the latest version that supports Datil
+- Specify the Datil network when instantiating Lit node clients from the SDK
```

Comment:
Can we add any more detail here on where to specify this in the SDK, or link to an article in the docs explaining where the user chooses the network?

Reply:
Addressed via [this commit](https://github.com/LIT-Protocol/docs/pull/306/commits/d732fc6add5fccb87e075f950a40dfef14c3953f)